frontend/tests/common/test_message.json

```diff
@@ -2,13 +2,13 @@
   "message": {
     "query_graph": {
       "nodes": {
-        "n1": {
-          "id": "MONDO:0005737",
-          "is_set": false
-        },
         "n0": {
-          "category": "biolink:Gene",
+          "categories": ["biolink:Gene"],
           "is_set": true
+        },
+        "n1": {
+          "id": ["MONDO:0005737"],
```

Comment:
In TRAPI 1.1, this should be "ids".

Reply:
We talked offline. The UI is currently between TRAPI 1.0 and 1.1, and this is a side effect. There will be a future PR that will transition us completely to 1.1.